recipes/_install-windows.rb

```diff
@@ -102,6 +104,8 @@ def unmute_host(context)
   # custom credentials and use them if that's the case.
   install_options.concat(' DDAGENTUSER_NAME=%DDAGENTUSER_NAME%') if ddagentuser_name
   install_options.concat(' DDAGENTUSER_PASSWORD=%DDAGENTUSER_PASSWORD%') if ddagentuser_password
+
+  install_options.concat(" NPM=#{dd_agent_install_npm}") unless dd_agent_install_npm.nil?
```

Comment:
```suggestion
  install_options.concat(" NPM=#{dd_agent_install_npm}") if dd_agent_install_npm
```
Actually this was correct before: We don't support passing `NPM=false` (it gets treated the same way as passing `true`)